egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -298,8 +313,9 @@ if [ $stage -le -4 ] && [ -z "$egs_dir" ]; then
   extra_opts+=(--transform-dir $transform_dir)
   extra_opts+=(--left-context $left_context)
   extra_opts+=(--right-context $right_context)
-  extra_opts+=(--valid-left-context $((chunk_width + left_context)))
-  extra_opts+=(--valid-right-context $((chunk_width + right_context)))
+  extra_opts+=(--valid-left-context $((minibatch_chunk_size + left_context)))
```

Comment:
OK. You might want to try tuning this value through experimentation. Please try using values like 
minibatch_chunk_size, minibatch_chunk_size * 2, minibatch_chunk_size * 3, minibatch_chunk_size * 4, minibatch_chunk_size * 5 ....